kfoldcv.py

```diff
@@ -0,0 +1 @@
+print "helloworld"
```

Comment:
Please change this to a Python 3 print statement.

Also, please add a space in between "hello" and "world"